resident/resident-service/src/main/java/io/mosip/resident/service/impl/ResidentServiceImpl.java

```diff
@@ -254,14 +246,27 @@ public class ResidentServiceImpl implements ResidentService {
 
 	@Value("${resident.service.unreadnotificationlist.id}")
 	private String unreadnotificationlist;
-	
+
+	private static String authTypes;
+
+	@Value("${auth.types.allowed}")
+	public void setAuthTypes(String authType) {
```

Comment:
No need to use @value on the setter, you can use it on the field.

Reply:
yes but I am using authTypes variable in a static function named AUTH_TYPE_FUNCTION and only static variables are allowed in this method. 	`private static String authTypes;` and if i put @value before this then it's taking null. so used @value before setter.